step-templates/redgate-create-database-release-worker-friendly.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
What changed in this file?

Reply:
The other Redgate Steps require you to reference a step that downloaded the package.  This will use the package reference feature introduced in 2018.7 for workers.